### ISSUE TITLE
Neverwinter (109600)

### ISSUE THREAD

**Issue by winiciuscota** [ValveSoftware/Proton#1725]
# Compatibility Report
- Name of the game with compatibility issues: Neverwinter
- Steam AppID of the game: 109600

## System Information
- GPU:  Intel Open Source Technology Center Mesa DRI Intel(R) Ivybridge Mobile x86/MMX/SSE2
- Driver/LLVM version: Mesa 18.2.2-1
- Kernel version: 4.18.7-rt5-MANJARO
- Link to full system information report: [steam-109600.log](https://github.com/ValveSoftware/Proton/files/2475154/steam-109600.log)
- Proton version: 18.7-8

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
There are no fonts on fonts. Tried installing corefonts with prontontricks and winetricks(and changing the prefix) but it didn't work. This game works fine on pure wine.

## Reproduction
Just get to the login screen

![image](https://user-images.githubusercontent.com/6284656/46902194-523e5380-ce97-11e8-9b14-e6c8a3cf2271.png)

**Comment by dreamer** [comment]
What version of wine are you using? Is bug present with PROTON_USE_WINED3D11 as well?

Can you do the following: run the game in normal wine, find it's pid, then `lsof -p <pid> > files.txt`, then repeat the same when running in Proton and upload results as gists to Github? It could help with pinpointing the issue.

**Comment by kisak-valve** [comment]
Hello @winiciuscota, `INTEL-MESA: warning: Ivy Bridge Vulkan support is incomplete` hints that your issue is from the incomplete vulkan support for mesa/Intel/Ivy Bridge. Adding `PROTON_USE_WINED3D11=1 %command%` to the game's launch options like @dreamer suggests should improve the behavior.

**Comment by winiciuscota** [comment]
Many thanks for the help.

I have wine 3.17

Using PROTON_USE_WINED3D11=1 %command% the game runs fine but the performance is much worse, I guess that when I tried this game 2 months ago on lutris it was using dx9 because the performance wasn't so bad, It was kind of playable. Is there a proton parameter to force the game to use dx9?

I'm using and thinkpad t430, so I guess the problem is really the incomplete intel drivers for ivy bridge. Is it still useful to get these logs @dreamer?

**Comment by dreamer** [comment]
@winiciuscota I was hoping to compare what libraries are used in both versions - we now know that issue lies in incomplete Vulkan support, therefore there's no point.

Do you see this problem happening in other titles or only Neverwinter?

> Is there a proton parameter to force the game to use dx9?

Yes. If there's no in-game switch, you can always force it with `PROTON_NO_D3D11=1`.

**Comment by winiciuscota** [comment]
Thanks @dreamer 

Neverwinter is the only dx11 game that I have tried on proton.

**Comment by kisak-valve** [comment]
[Whitelist] Neverwinter (109600) 

Issue transferred from https://github.com/ValveSoftware/Proton/issues/2089.
@vampywiz17 posted on 2018-12-14T18:19:43:

# Whitelist Request
- Name of the game to be whitelisted: Neverwinter
- Steam AppID of the game: 109600

## System Information
- GPU: GTX1060 3GB
- Driver/LLVM version: nvidia 390.87
- Distro version: Ubuntu 18.10
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/vampywiz17/5711eb79a513b9157773b2691b07bd41
- Proton version: 3.16-5

## I confirm:
- [ x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [ ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [ ] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [ x] Although I consider the gaming experience equal to Windows there are remaining issues:

Only one thing. Need to set mouse to 125hz to resolve input lag. No other problem at all.

**Comment by GloriousEggroll** [comment]
Just a small FYI the launcher for neverwinter is affected by the high core count bug mentioned here:
https://github.com/ValveSoftware/wine/issues/35

it does not load properly without this on high core count machines

**Comment by kisak-valve** [comment]
Neverwinter works with Wine but not with Steam Play

Issue transferred from https://github.com/ValveSoftware/Proton/issues/2411.
@FinixFighter posted on 2019-03-12T20:22:11:

# Compatibility Report
- Name of the game with compatibility issues: Neverwinter
- Steam AppID of the game: 109600

## System Information
- GPU: AMD KABINI
- Driver/LLVM version: 4.4 Mesa 18.2.2
- Kernel version: 4.15.0-45-generic
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: 3.16-8 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.


[steam-109600.log](https://github.com/ValveSoftware/Proton/files/2958645/steam-109600.log)


## Symptoms 
The game works perfectly with Wine, but it doesn't start with Steam Play. This is a screenshot of the error:
![Neverwinter error](https://user-images.githubusercontent.com/19877271/54233242-d0695980-450c-11e9-8580-5345d4884e9a.png)



## Reproduction
I simply click the "Play" button on Steam (in my library), then the first game's window loads, I log in with my account and then I click "Play". Then I get the error.

**Comment by kisak-valve** [comment]
Hello @FinixFighter, can you verify you have Vulkan support for your video card installed with something like `apt policy mesa-vulkan-drivers mesa-vulkan-drivers:i386`?

**Comment by kisak-valve** [comment]
You'll also want to give https://github.com/ValveSoftware/Proton/wiki/For-AMD-users-having-issues-with-non-OpenGL-games a read.

**Comment by FinixFighter** [comment]
By running "apt policy mesa-vulkan-drivers mesa-vulkan-drivers:i386" I get this:
`mesa-vulkan-drivers:
  Installed: (none)
  Candidate: 18.2.2-0ubuntu1~18.04.2
  Version table:
     18.2.2-0ubuntu1~18.04.2 500
        500 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages
     18.0.0~rc5-1ubuntu1 500
        500 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages
mesa-vulkan-drivers:i386:
  Installed: (none)
  Candidate: 18.2.2-0ubuntu1~18.04.2
  Version table:
     18.2.2-0ubuntu1~18.04.2 500
        500 http://archive.ubuntu.com/ubuntu bionic-updates/universe i386 Packages
     18.0.0~rc5-1ubuntu1 500
        500 http://archive.ubuntu.com/ubuntu bionic/universe i386 Packages
`
It seems I don't have vulkan, right? But, since the game works perfectly with Wine, shouldn't it work on Steam too?

**Comment by kisak-valve** [comment]
By default, Proton uses DXVK to translate D3D10 and D3D11 to Vulkan. You can add `PROTON_USE_WINED3D=1 %command%` to the game's launch options to have it use OpenGL for games using those graphics APIs.

**Comment by FinixFighter** [comment]
Yes, game works perfectly by adding `PROTON_USE_WINED3D=1 %command%` to the game's launch options!

**Comment by jicama** [comment]
I was getting a blank black screen with the proprietary Nvidia driver, but PROTON_USE_WINED3D=1 fixed that, too.

**Comment by jicama** [comment]
My problem turned out to be that on my Optimus laptop, Vulkan was trying and failing to use the integrated Intel graphics.  Removing the intel vulkan icd made it work.  This was an issue for all Vulkan programs, not specific to Neverwinter or even DXVK.

**Comment by jicama** [comment]
> Only one thing. Need to set mouse to 125hz to resolve input lag. No other problem at all.

This seems to be a problem with many games, as in issue #2455 .

**Comment by txtsd** [comment]
Runs with Proton 5.0-10. Does not go past the Neverwinter loading screen with Proton 5.13-1.

**Comment by kisak-valve** [comment]
Hello @txtsd, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by txtsd** [comment]
Surprisingly, it worked with Proton 5.13-1 this time. I think I had skipped the vulkan shader generation the last time around; I'm not sure if that causes any problems. It even works without `PROTON_USE_WINED3D=1` now. With wined3d, there is extreme input lag from the mouse, so I can play better now that that's gone.

Pardon the false alarm.
